test/passes/fannkuch3.bin.txt

```diff
@@ -183,6 +183,485 @@ Abbrev table for offset: 0x00000000
                  [0x00000003, 0x0000039a)
                  [0x0000039c, 0x000006e2))
 
+0x00000026:   DW_TAG_subprogram [2] *
+                DW_AT_name [DW_FORM_strp]	( .debug_str[0x000000fa] = "free")
+                DW_AT_decl_file [DW_FORM_data1]	("/usr/local/google/home/azakai/Dev/emscripten/system/include/libc/stdlib.h")
```

Comment:
these paths are embedded in the test binary and won't change when we run them on another machine, right?

Reply:
Correct, yes, these are embedded paths.

Now that you mention it I'm surprised these paths are absolute like that, not sure why clang does it that way.